tutorial/game/tutorial_screen_displayables.rpy

```diff
@@ -470,7 +470,7 @@ label button_displayables:
                         text _("Heal") style "button_text" yalign 0.5
                         bar value AnimatedValue(health, 100, 1.0) yalign 0.5 xsize 200
 
-    e "A button takes another displayable as children. Since that child can be a layout, it can takes as many children as you want."
+    e "A button takes another displayable as children. Since that child can be a layout, it can take as many children as you want."
```

Comment:
"another" (singular) also doesn't go well with "children" (plural)

Reply:
Agreed, I was only changing the most blatant errors since I didn't intend to actually modify any of the content, but I've added that fix with 63ba851. I turned it singular in the first sentence since the next sentence refers to "that child" (singular).